webapp-django/crashstats/crashstats/views.py

```diff
@@ -188,14 +188,47 @@ def report_index(request, crash_id, default_context=None):
     context["public_raw_keys"].sort(key=lambda s: s.lower())
     context["protected_raw_keys"].sort(key=lambda s: s.lower())
 
+    data_urls = []
```

Comment:
I switched link creation to something that's more predictable and uses the existing API.